nixopsgce/backends/gce.py

```diff
@@ -66,7 +66,8 @@ def parse_block_device(xml):
                 'disk': self.get_option_value(xml, 'disk', 'resource', optional = True),
                 'disk_name': opt_disk_name(self.get_option_value(xml, 'disk_name', str, optional = True)),
                 'snapshot': self.get_option_value(xml, 'snapshot', str, optional = True),
-                'image': self.get_option_value(xml, 'image', 'resource', optional = True),
+                'image': self.get_option_value(xml, 'image', str, optional = True),
+                'publicImageProject' : self.get_option_value(xml, 'publicImageProject', str, optional = True),
```

Comment:
What happens if this is `None`(null)?

Reply:
This must be optional because in case it has value of `None`, the old method would be used consisting in creating the disk from our own bootstrap image

```                # Retrieve GCENodeImage based on family name and project
                if v['publicImageProject']:
                    try:
                        img = self.connect().ex_get_image_from_family(
                                  image_family=v['image'],
                                  ex_project_list=[v['publicImageProject']],
                                  ex_standard_projects=False,
                              )
               .......................
                .........................
                else:
                    img = v['image']